quantum/process_keycode/process_unicode_common.c

```diff
@@ -136,20 +136,10 @@ __attribute__((weak)) void unicode_input_cancel(void) {
     set_mods(unicode_saved_mods);  // Reregister previously set mods
 }
 
-__attribute__((weak)) uint16_t hex_to_keycode(uint8_t hex) {
```

Comment:
Did you mean to remove this? 

I think that this still needs to be there.

Reply:
Yes, it already exists in quantum.c. I tested it and it still works :+1:

Reply:
Ah, okay. Just making sure.